### ISSUE TITLE
[cygnus-common][API] Refactor: merge "v1" and "not-v1" documents

### ISSUE THREAD

**Issue by frbattid** [telefonicaid/fiware-cygnus#1367]
Currently, there are a couple of documents about the API:

* A document for the "v1" part of the API.
* A document fot the "not-v1" part of the API, which really are aliases of operations within the first part.

Both documents have to be merged in a single one, describing the "not-v1" operations as aliases of "v1" operations.

**Comment by fgalan** [comment]
By "not-v1" you mean "v2" in operations related with NGSIv2?

**Comment by frbattid** [comment]
No, it is not about NGSI versions. It is is about Cygnus API versions: some operations, particularly those not requested by IoTP team, have routes such as "/v1/namemappings", while other operations, particularly those requested by IoTP, are of type "/admin/log". The good thing is all the "not-v1" operations have a "v1" counterpart; on the contrary, not all the "v1" operations have a "not-v1" counterpart, but that's not a problem.
